src/Frontend/Path.hs

```diff
@@ -106,7 +106,7 @@ import Types
 
 -- * types
 
--- FIXME: Introduce AllowGet only
+-- FIXME: Introduce AllowGet only, or better make it a list of [GET|POST].
```

Comment:
[GET, POST] is isomorphic to [POST|GET].
What is the interpretation for types like: [GET], [GET, GET]